### PR TITLE
refactor: align configCases runner

### PR DESCRIPTION
**Pull request by h-a-n-a**
## Summary

closes https://github.com/web-infra-dev/rspack/issues/5686

## Require Documentation?

- [ ] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

**Comment by h-a-n-a** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #5620** <a href="https://app.graphite.dev/github/pr/web-infra-dev/rspack/5620?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/web-infra-dev/rspack/5620?utm_source=stack-comment).

File: packages/rspack/tests/configCases/builtin-swc-loader/styled-components/index.jsx [hunk 1]
```diff
@@ -2,11 +2,9 @@ import { Container } from "./Button";
 import { Container as Container2 } from "./Button2";
 
 it("styled components", () => {
-	expect(Container.displayName).toMatch("Button__Container");
```
Comment (SyMind) [inline packages/rspack/tests/configCases/builtin-swc-loader/styled-components/index.jsx:5]: Why was it removed.

Comment (h-a-n-a) [reply packages/rspack/tests/configCases/builtin-swc-loader/styled-components/index.jsx:5]: Webpack test runner is running in mode production by default in which it does not have this field.

Comment (h-a-n-a) [reply packages/rspack/tests/configCases/builtin-swc-loader/styled-components/index.jsx:5]: Webpack test runner is running in mode production by default in which it does not have this field.

**Review by SyMind — APPROVED** [review]

**Pull request merged**